packages/renderer/src/stores/agent-workspace-terminal-store.ts

```
@@ -0,0 +1,64 @@
+      const index = list.indexOf(terminal);
+      if (index > -1) {
+        list.splice(index, 1);
+      }
+      return list;
+    });
+  }
+});
+
+export function registerTerminal(terminal: TerminalOfAgentWorkspace): void {
+  agentWorkspaceTerminals.update(list => {
+    list = list.filter(t => t.workspaceId !== terminal.workspaceId);
```

Biome (2.4.9)

[error] Assigning a function parameter is confusing.

(lint/style/noParameterAssign)